frontend/src/components/stages/stage_footer.ts

```diff
@@ -41,16 +42,11 @@ export class Footer extends MobxLitElement {
       this.isLoadingNext = true;
       // Handle custom onNextClick
       await this.onNextClick();
+      
       // Save last stage and mark experiment as completed
       await this.participantService.completeLastStage();
-      // Navigate to landing
-      this.routerService.navigate(
-        Pages.PARTICIPANT,
-        {
-          'experiment': this.routerService.activeRoute.params['experiment'],
-          'participant': this.routerService.activeRoute.params['participant'],
-        }
-      );
+      await this.participantService.routeToEndExperiment(ParticipantStatus.SUCCESS);
```

Comment:
Just to confirm: we want to route participants to home (which will show them a 403 error since they don't have access) rather than the landing page for their current experiment (where they can still view the answers they submitted)?